admin/commands/config.go

```diff
@@ -59,7 +59,7 @@ type ConfigCommand struct {
 	LogLinesCount     uint     `help:"Take and return N most recent log lines in logs.zip for each: server, every configured exporters and agents" default:"1024"`
 }
 
-func (cmd *ConfigCommand) args(globals *flags.GlobalFlags) (res []string, switchedToTLS bool) {
+func (cmd *ConfigCommand) args(globals *flags.GlobalFlags) (res []string, switchedToTLS bool) { //nolint:nonamedreturns
```

Comment:
I agree, most of the functions could be refactored to obey the rule. However, I was reluctant to do so, since some of the refactored functions would require testing, which is never in the scope of such PRs.

Reply:
I think we can refactor easy cases and skip QA, automated tests should be enough. 

@BupycHuk  any concerns?

Reply:
No concerns

Reply:
OK, will do wherever it is a no-brainer )